org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/yaml/YAMLLanguageServer.java

```diff
@@ -141,6 +143,15 @@ public void handleMessage(Message message, LanguageServer languageServer, URI ro
 				}
 			}
 		});
+		
+		IConfigurationElement[] conf = Platform.getExtensionRegistry().getConfigurationElementsFor(JSonLanguageServer.SCHEMA_EXT);
```

Comment:
We could try to extract a "JSonSchemaRegistry" or something like that to more easily share this code between json and yaml.
(Just food for thought, I'm not asking you to change it now).